src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -34,6 +34,36 @@ module Proof_level = struct
   let compiled = of_string compiled
 end
 
+module Constraint_constants = struct
```

Comment:
Can you stick a comment above here that explains something like "this is for storing variables that affect the generation of proving keys"?